test/unit/test_train.py

```diff
@@ -87,19 +54,28 @@ def test_build_callbacks():
         assert isinstance(callback, tf.keras.callbacks.Callback)
 
 
-def test_train():
+def test_train_and_predict():
```

Comment:
should this maybe test if it produces an output in the logs instead of just if it is able to run?

Reply:
You mean, test if there's an output folder?
For `train`, no need as the `predict` will have to use the checkpoint, for `predict` ok, i can check this ;)